tests/base.py

```diff
@@ -15,4 +15,8 @@ def setUp(self):
         self.db = self.loop.run_until_complete(db_co)
 
     def tearDown(self):
-        self.loop.run_until_complete(self.db.flushall())
+        async def delete_all():
+            async for k in self.db.iscan(match='*Test*', count=100):
```

Comment:
So if there are > 100 keys then only the first 100 get deleted? That sounds like a bug.